### ISSUE TITLE
Accessibility updates

### ISSUE THREAD

**Issue by acobbb** [deejay-hub/timeline-lwc#305]
**Is your feature request related to a problem? Please describe.**
We just started using the Time Warp app at the University of Arizona, and after an accessibility review we discovered some issues around accessibility. The elements on the timeline are not available to a screen reader, and are not tabable for a user who is only using a keyboard and not a mouse. The date filters are also not accessible to these users.

**Describe the solution you'd like**
We are in the early discovery stages of how to solve these issues, but if you have any suggestions on solutions or approaches we would love your input!

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by deejay-hub** [comment]
These are important updates but will need help from the community to suggest what needs to be done and how to do it. If there's anyone with experience that can guide the conversation feel free to comment.

**Comment by acobbb** [comment]
@its-me-mario is our UX designer. He is out of the office until next week, but should be able to provide some more details on the specifics of what came out of the accessibility review.

**Comment by acobbb** [comment]
@deejay-hub Thanks for incorporating our updates into the next release! Do you have any timeline on when Epsilon is expected to be released?

**Comment by deejay-hub** [comment]
If you mean packaged and put on AppExchange then we don't have a date. We work on this in our spare time and we've focused on getting the code base..here... up to date.

**Comment by deejay-hub** [comment]
v1.15 is available - but issues with the date filter for zooming meant that we had to remove that feature. So tab stops made it, screen reader functionality works but you can't modify the dates used to zoom yet.

[AppExchange Link](https://login.salesforce.com/packaging/installPackage.apexp?p0=04t2w000009gtDFAAY)

**Comment by acobbb** [comment]
Thanks so much! What were the issues with the date zooming? Would love to fix the issues and get that functionality back in

**Comment by deejay-hub** [comment]
We kept the branch called a11y with a version of the code for your reference you can still access. We may have broken your original code but in this version we saw the following with a cursory test.

- Changing dates just moved the zoom to the start of the time range with a NaN error
- Changing the dates didn't seem to help the zoom gets stuck
- Looks like there was no anticipation that a user might select a start or end beyond the plotted range
- No date picker in the input preventing data types other than a valid date
- Applying didn't close the filter
- Refreshing causes an error

Also from a design perspective it might be better to use the existing space in the top right to have the filter. We currently have 'Showing: Date 1 - Date 2'. We could just replace the date labels with inputs to make it more intuitive and faster to locate. A user just has to overtype what is in the new inputs.

More than happy to get a new pull request based on the current version if you have the time to look into it.

![Image](https://github.com/user-attachments/assets/e8ca7eae-0a41-46f8-b1f7-11016d740609)

**Comment by deejay-hub** [comment]
Closing due to inactivity
